### DISCUSSION TITLE
How to keep formatting between lines?

### DISCUSSION THREAD

**Discussion by nickschinestzki** [facebook/lexical#5619] [Q&A] [answered]
I'm trying to achieve a different behavior regarding the formatting between lines in the Lexical Editor.

### The current behavior
1. In an empty editor, the user starts adding formatting and styles in the editor (font-family, font-size, color, bold, italic, etc)
2. When you press `Enter` once, it will insert a new line, which will have an "unstyled cursor" but upon new typing, the styles and formatting are applied again to the text.
3. When you press `Enter` twice, the new line, separated by the other ones, will lose all of the previous formatting and styling, returning to the default. 

https://github.com/facebook/lexical/assets/33634787/b7fc7e60-0ce6-4d5c-be79-bea582a9a675

### The desired behavior
1. In an empty editor, the user starts adding formatting and styles in the editor (font-family, font-size, color, bold, italic, etc)
2. When pressing `Enter` regarding of how much times/lines, the styles and formats are preserved. The user doesn't have to select the font-family and font-size again (for example) every time he creates a new a paragraph.

How can I achieve this?

**Comment by ivailop7** [comment, marked as answer]
This should be converted into an issue and be fixed.
